project/Rat.scala

```diff
@@ -114,7 +114,8 @@ object Rat {
     file("daffodil-sapi/src/test/resources/test/sapi/myData16.dat"),
     file("daffodil-sapi/src/test/resources/test/sapi/myData19.dat"),
     file("daffodil-sapi/src/test/resources/test/sapi/myDataBroken.dat"),
-    file("daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/parse_int32"),
+    file("daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/ex_int32_parse"),
+    file("daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/orion_command_parse"),
```

Comment:
Can you add .bin, .dat or something just so it makes it more clear that these are binary files? Makes it more obvious the reason why we need to exclude them from rat.

Also, can these runtime-2 excludes be sorted so they are before daffodil-sapi?